frontend/src/employee-mobile-frontend/child-attendance/ChildListItem.tsx

```diff
@@ -115,9 +115,10 @@ export default React.memo(function ChildListItem({
   }>()
   const { selectedGroupId } = useSelectedGroup()
 
-  const { data: groupNotes } = useQuery(groupNotesQuery(child.groupId ?? ''), {
-    enabled: child.groupId != null
+  const rawGroupNotes = useQueryResult(groupNotesQuery(child.groupId ?? ''), {
+    enabled: !!child.groupId
   })
+  const groupNotes = child.groupId ? rawGroupNotes.getOrElse([]) : []
```

Comment:
It seems to me that this jus a longer way to do this:
```
  const { data: groupNotes = [] } = useQuery(groupNotesQuery(child.groupId ?? ''), {
    enabled: !!child.groupId
  })
```

When query is disabled or fetching, `useQuery` returns `undefined` `data` and the default `[]` kicks in.